### ISSUE TITLE
[BUG] Connecting with USB causes all the buttons and everything on the controller out of control

### ISSUE THREAD

**Issue by RaigaX9** [Davidobot/BetterJoy#762]
**Describe the bug**
When I'm connecting to the USB, when I open up Dolphin and sometimes ePSxe, the controller is going chaotic for some reason. Here is what's happening here below for example on Dolphin:
![out-of-controller-pro](https://user-images.githubusercontent.com/2623719/115975311-a7ac4e80-a531-11eb-8eac-bf1b2947a791.gif)

The version of BetterJoy I'm using is v6.4.

**Expected behavior**
It should not go haywire based on the screenshot

**Controller (please complete the following information):**
 - Type: Pro Controller
 - Connection: USB

**Comment by Davidobot** [comment]
Well, duh, select the xinput device to get in out from, not the dinput one.

**Comment by RaigaX9** [comment]
Ohhh ok. It was weird I actually did the XInput before and it didn't work. I just did it again and again it didn't work. I was confused as to why that is so I had recently downloaded a driver updating software to make sure. Turns out that the bus ports were 6 years outdated and after updating that along with all of my other drivers as well as running Betterjoy, it looks like that works and all is good. Thanks and sorry about that.
